doc/languages-frameworks/haskell.section.md

```diff
@@ -929,10 +929,12 @@ for this to work.
   Haskell dependencies, but will still link dynamically against libc, GMP and
   other system library dependencies.
 
-  If the library being built or its dependencies use their Cabal-generated
+  If a library or its dependencies use their Cabal-generated
```

Comment:
IIRC the doc team recommends a single line per sentence, so that we don't have to reflow. I haven't done the reflowing to keep the diff readable.